src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -276,15 +285,15 @@ let printMessages (msgs: AnalyzerMessage list) =
             m.Range.FileName,
             m.Range.StartLine,
             m.Range.StartColumn,
-            (m.Severity.ToString()),
+            m.Severity.ToString(),
             m.Code,
             m.Message
         )
     )
 
     ()
 
-let writeReport (results: AnalyzerMessage list option) (codeRoot: string option) (report: string) =
+let writeReport (results: AnalyzerMessage list) (codeRoot: string option) (report: string) =
```

Comment:
This was never actually optional; we always statically knew that this `option` was `Some`. I tightened up the domain by representing that fact in the types.